vowpalwabbit/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ add_library(vw_io STATIC io/io_adapter.h io/io_adapter.cc)
 target_link_libraries(vw_io PRIVATE ZLIB::ZLIB)
 add_library(VowpalWabbit::io ALIAS vw_io)
 
+add_subdirectory(parser/flatbuffer)
+add_library(FlatbuffersTarget INTERFACE)
```

Comment:
Does this need to exported? Does it affect the install target? (does `cmake -DVW_INSTALL=On` still work?)

Reply:
`cmake -DVW_INSTALL=On` still works